sdk/template/CMakeLists.txt

```
@@ -9,3 +9,4 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 
 add_subdirectory(azure-template)
+add_subdirectory(azure-template-second)
```

I don't have a strong preference on the name.  Maybe `engsys-testpkg` or `azure-engsys-testpkg`.  Something to make clear this package in about validating automation and should not be cloned as the starting point for client library authors.

```suggestion
add_subdirectory(azure-template-second)

```